payjoin/src/core/receive/common/mod.rs

```diff
@@ -65,11 +30,11 @@ pub use v1::*;
 /// Call [`Self::commit_outputs`] to proceed.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct WantsOutputs {
-    original_psbt: Psbt,
-    payjoin_psbt: Psbt,
-    params: Params,
-    change_vout: usize,
-    owned_vouts: Vec<usize>,
+    pub(crate) original_psbt: Psbt,
+    pub(crate) payjoin_psbt: Psbt,
+    pub(crate) params: Params,
+    pub(crate) change_vout: usize,
+    pub(crate) owned_vouts: Vec<usize>,
```

Comment:
Because tests are still tangled across the common/v1 modules, this visibility change is required. It's addressed in a later commit which untangles the tests.